arrays/arrays.c

```diff
@@ -79,10 +84,10 @@ char *arr_read(Array *arr, int index)
 {
 
     // Throw an error if the index is greater or equal to than the current count
-    if (index > arr->count)
+    if (index >= arr->count)
     {
         fprintf(stderr, "Index %i out of range\n", index);
-        return -1;
+        return NULL;
```

Comment:
:) I like to look at the tests from the very start. they give great hints about what's expected from your functions